src/pages/billing/Project.js

```diff
@@ -43,37 +60,39 @@ const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ sort,
   ])
 })
 
-const rowBase = { display: 'flex', alignItems: 'center', width: '100%' }
-
-const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details, additionalInfo }) => {
+const ExpandedInfoRow = Utils.memoWithName('ExpandedInfoRow', ({ title, details, errorMessage }) => {
   const expandedInfoStyles = {
```

Comment:
I'd encourage you to consider inlining these. Having them separate doesn't seem to add much.